app/adapters/tsheets_adapter.rb

```diff
@@ -0,0 +1,11 @@
+class TsheetsAdapter
+  def initialize
+    @tsheets_api = TSheets::API.new do |config|
+      config.access_token = ENV['TSHEETS_ACCESS_KEY']
```

Comment:
Good point, the benefits may not be great as you say, but I don't think it's bad practice to restrict the use of ENV to initializers. Secrets is probably the way to go in this case. :-)

Nice catch on the `fetch` too.